tests/subsys/logging/log_core_additional/testcase.yaml

```diff
@@ -5,3 +5,8 @@ tests:
   logging.add.sync:
     tags: logging
     extra_args: CONF_FILE=log_sync.conf
+  logging.add.user:
+    tags: logging
+    platform_exclude: mps2_an521 mps2_an385
```

Comment:
Is there any reason to exclude them?

Reply:
updated. build error occurs, I've reported an issue